rules/no-duplicate-literal-elements.js

```diff
@@ -0,0 +1,110 @@
+'use strict';
+const MESSAGE_ID = 'no-duplicate-literal-elements';
+const messages = {
+	[MESSAGE_ID]: 'Hardcoded `{{type}}` of `{{value}}` with duplicates.',
+};
+
+const arraySelector = [
+	'VariableDeclarator',
+	' > ',
+	'ArrayExpression',
+	' , ',
+	'ExpressionStatement',
+	' > ',
+	'ArrayExpression',
+	' , ',
+	'AssignmentExpression',
+	' > ',
+	'ArrayExpression',
+].join('');
+
+const setSelector = [
+	'NewExpression',
+	'[arguments.0.type="ArrayExpression"]',
+	'[callee.name="Set"]',
+	' > ',
+	'ArrayExpression',
+].join('');
```

Comment:
You can use `newExpressionSelector` from `rules/selectors/index.js`, it's stricter than this.

Reply:
Thanks, I will go to confirm